src/main/java/org/liquidengine/legui/style/Style.java

```diff
@@ -0,0 +1,309 @@
+package org.liquidengine.legui.style;
+
+import org.joml.Vector2f;
+import org.joml.Vector4f;
+import org.liquidengine.legui.color.ColorConstants;
+import org.liquidengine.legui.font.Font;
+import org.liquidengine.legui.style.border.SimpleLineBorder;
+
+/**
+ * The type Style.
+ *
+ * @author Aliaksandr_Shcherbin.
+ */
+public class Style {
+
+    private Background background = new Background();
+    private Border border = new SimpleLineBorder(ColorConstants.gray(), 1);
+    private Font font;
+
+    private float topLeftCornerRadius;
+    private float topRightCornerRadius;
+    private float bottomLeftCornerRadius;
+    private float bottomRightCornerRadius;
```

Comment:
Is this really needed? why not use a single Vector4f?

Reply:
Not really needed, could be replaced.